app/javascript/lib/validation/validate-file.js

```diff
@@ -49,6 +50,13 @@ function validateFileName(file, studyFile, allStudyFiles, allowedFileExts=['*'])
  * @param allowedFileExts { String[] } array of allowable extensions, ['*'] for all
  */
 async function validateLocalFile(file, studyFile, allStudyFiles=[], allowedFileExts=['*']) {
+
+  // if clientside file validation feature flag is false skip validation
+  const flags = getFeatureFlagsWithDefaults()
+  if (!flags?.clientside_validation) {
```

Comment:
Blocker: this will skip CSFV if this flag does not exist here, which seems feasible inadvertently.  We should be more robust by skipping CSFV if and only if this flag _exists_ and is false.

```suggestion
  if (clientside_validation in flags && !flags.clientside_validation) {
```

For reference:
```
foo = {'a': true, 'b': false}
if (!foo?.clientside_validation) {console.log('CSFV flag is missing, skipping CSFV')}
CSFV flag is missing, skipping CSFV
if ('clientside_validation' in foo && !foo.clientside_validation) {console.log('CSFV flag is missing, skipping CSFV')}
undefined
```

This would also let us de-noise test code.